service/app/commands/message_buffer.go

```diff
@@ -128,6 +128,15 @@ func (m *MessageBuffer) persistTransaction(adapters Adapters, feedsToPersist fee
 		authorRef := firstMessage.Author()
 		feedRef := firstMessage.Feed()
 
+		feedIsBanned, err := adapters.BanList.ContainsFeed(feedRef)
```

Comment:
Needs tests but right now the whole buffer doesn't really have tests. They will be added in #44.